Rakefile

```diff
@@ -23,20 +23,20 @@ end
    config.ignore_paths = ["tests/*.pp", "spec/**/*.pp", "pkg/**/*.pp"]
  end
 
-task :default => [:test]
-
-# The acceptance tests for Registry are written in standard beaker format however
-# the preferred method is using beaker-rspec.  This rake task overrides the 
-# default `beaker` task, which would normally use beaker-rspec, and instead
-# invokes beaker directly.  This is only need while the module tests are migrated
-# to the newer rspec-beaker format
-task_exists = Rake.application.tasks.any? { |t| t.name == 'beaker' }
-Rake::Task['beaker'].clear if task_exists
-desc 'Run acceptance testing shim'
-task :beaker do |t, args|
-  beaker_cmd = "beaker --options-file acceptance/.beaker-pe.cfg --hosts #{ENV['BEAKER_setfile']} --tests acceptance/tests --keyfile #{ENV['BEAKER_keyfile']}"
-  Kernel.system( beaker_cmd )
-end
+# task :default => [:test]
```

Comment:
Can delete these instead of commenting.  Also could revert https://github.com/puppetlabs/puppetlabs-registry/commit/a168bd45db3ed492aace01b48d385b52fdd4f1c5

Reply:
Yup, forgot about that!